### PR TITLE
Removed Rancher CLI from QS; added to main CLI page for 2.5, 2.6

### LINKED ISSUE #4098: [Rancher2] CLI docs are split up
We had some confusion on what auth providers were supported when logging in with CLI, that isn't stated here: https://rancher.com/docs/rancher/v2.5/en/cli/#cli-authentication

But oddly it is stated in the quick start guide: https://rancher.com/docs/rancher/v2.6/en/quick-start-guide/cli/

I believe that whole bottom section in the quick start should be moved to the main CLI page. Another option is to move the quick start page entirely to the CLI page, I'm not sure it really belongs in the "deployment" section.

### PR DESCRIPTION
**Pull request by jtravee**
Closes #4098

### CODE REVIEW

File: content/rancher/v2.5/en/cli/_index.md [hunk 1]
```diff
@@ -80,3 +78,41 @@ The following commands are available for use in Rancher CLI.
...
 All commands accept the `--help` flag, which documents each command's usage.
+
+### Limitations
+
+The Rancher CLI **cannot** be used to install [dashboard apps or Rancher feature charts](../helm-charts/).
+
+# Using the CLI with Rancher
+
+Interact with Rancher using command line interface (CLI) tools from your workstation.
+
```
Comment (btat) [inline content/rancher/v2.5/en/cli/_index.md:89]: This should be used as the intro/description at the start of the page.

**Review by btat — CHANGES_REQUESTED** [review]
The new content being moved into this page is about CLI in general (Rancher CLI + kubectl), but the existing page is only about the Rancher CLI.  Rather than just adding the content to the end the page needs to be rearranged so that there's two major headers (one for Rancher CLI and other for kubectl).

File: content/rancher/v2.6/en/cli/_index.md [hunk 2]
```diff
@@ -88,3 +88,37 @@ All commands accept the `--help` flag, which documents each command's usage.
...
+
+_Requirements_
+
+If admins have [enforced TTL on kubeconfig tokens]({{<baseurl>}}/rancher/v2.6/en/api/api-tokens/#setting-ttl-on-kubeconfig-tokens), the kubeconfig file requires the [Rancher CLI](../cli) to be present in your PATH when you run `kubectl`. Otherwise, you’ll see an error like: 
+`Unable to connect to the server: getting credentials: exec: exec: "rancher": executable file not found in $PATH`. 
+
+This feature enables kubectl to authenticate with the Rancher server and get a new kubeconfig token when required. The following auth providers are currently supported: 
+
+1. Local
+2. Active Directory
```
[the remark this answers is not in the document]
Comment (jtravee) [reply content/rancher/v2.6/en/cli/_index.md]: Revised as suggested.

File: content/rancher/v2.5/en/cli/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.5/en/cli/_index.md]: Reorganized as discussed, thanks!

**Review by btat — APPROVED** [review]
lgtm

**Review by maxsokolovsky — APPROVED** [review]

**Pull request merged**